install_config/install/docker_registry.adoc

```diff
@@ -80,19 +80,20 @@ endif::[]
 
 === Deploying the Registry as a DaemonSet
 
-You may use the `oadm registry` command to deploy the registry as a
-link:http://kubernetes.io/docs/admin/daemons/[DaemonSet] with
-the `--daemonset` option.
+Use the `oadm registry` command to deploy the registry as a DaemonSet with the
+`--daemonset` option.
 
-A link:http://kubernetes.io/docs/admin/daemons/[DaemonSet] will run a pod
-on every node.  For more information on
-link:http://kubernetes.io/docs/admin/daemons/[DaemonSets] please see the
+Daemonsets ensure that when nodes are created, they contain copies of a
+specified pod. When the nodes are removed, the pods are garbage collected.
+
+For more information on DaemonSets, see the
 link:http://kubernetes.io/docs/admin/daemons/[DaemonSet documentation].
 
 [NOTE]
 ====
-Updating of DaemonSets is still a work in progress.
-It is recommended that a DeploymentConfig be used to deploy the registry.
+Currently, using DaemonSets with {product-title} is a work in progress, and the
+only supported process is creating a registry. Using deployment configurations
+to deploy a registry might be ideal.
```

Comment:
I think I'd make this a little more scary and recommend using the DC.  We don't want to recommend running as a DaemonSet until the upstream work is completed.